backend/core/src/email/email.service.ts

```diff
@@ -51,7 +51,7 @@ export class EmailService {
     }
     await this.send(
       user.email,
-      "Welcome to Bloom",
+      "Welcome",
```

Comment:
We can reference the existing translation location (https://github.com/bloom-housing/bloom/blob/b3da8965eb481a5f6ce093fa1eefa3b8af4d2fb1/backend/core/src/migration/1620660845209-seed-translation-entries.ts#L51) here as we are doing for the email we send in the confirmation fxn in this file

Reply:
Oh I'll take a look at that, thanks @emilyjablonski!